app/views/user_mailer/welcome.html.haml

```diff
@@ -1,25 +1,76 @@
+      = render 'application/mailer/checklist', key: 'share_step', title: t('user_mailer.welcome.share_title'), text: t('user_mailer.welcome.share_step'), checked: false, button_text: t('user_mailer.welcome.share_action'), button_url: web_url('start/share')
+      = render 'application/mailer/checklist', key: 'apps_step', title: t('user_mailer.welcome.apps_title'), text: t('user_mailer.welcome.apps_step'), checked: false, show_apps_buttons: true
+%table.email-w-full{ cellspacing: 0, cellpadding: 0, border: 0, role: 'presentation' }
+  %tr
+    %td.email-body-columns-td
+      .email-desktop-flex
+        /[if mso]
+          <table border="0" cellpadding="0" cellspacing="0" align="center" style="width:100%;" role="presentation"><tr><td style="width:50%; vertical-align:top;">
+        .email-desktop-column
+          %table.email-w-full{ cellspacing: 0, cellpadding: 0, border: 0, role: 'presentation' }
+            %tr
+              %td.email-column-td
+                %h2.email-h2= t('user_mailer.welcome.follows_title')
+                %p.email-h-sub= t('user_mailer.welcome.follows_subtitle')
+                = render partial: 'application/mailer/follow', collection: @suggestions
+                %table.email-w-full{ cellspacing: 0, cellpadding: 0, border: 0, role: 'presentation' }
+                  %tr
+                    %td.email-column-action-td
+                      = link_to '', href: web_url('explore/suggestions'), class: 'email-link-with-arrow' do
+                        = t('user_mailer.welcome.follows_view_more')
+                        %span= '❯'
+        /[if mso]
+          </td><td style="width:50%; vertical-align:top;">
+        .email-desktop-column
+          %table.email-w-full{ cellspacing: 0, cellpadding: 0, border: 0, role: 'presentation' }
+            %tr
+              %td.email-column-td
+                %h2.email-h2= t('user_mailer.welcome.hashtags_title')
+                %p.email-h-sub= t('user_mailer.welcome.hashtags_subtitle')
+                = render partial: 'application/mailer/hashtag', collection: @tags
+                %table.email-w-full{ cellspacing: 0, cellpadding: 0, border: 0, role: 'presentation' }
+                  %tr
+                    %td.email-column-action-td
+                      = link_to '', href: web_url('explore/tags'), class: 'email-link-with-arrow' do
+                        = t('user_mailer.welcome.hashtags_view_more')
+                        %span= '❯'
+        /[if mso]
+          </td></tr></table>
+.email-extra-wave
+%table.email-w-full{ cellspacing: 0, cellpadding: 0, border: 0, role: 'presentation' }
+  %tr
+    %td.email-extra-td
+      = render 'application/mailer/feature', key: 'feature_control', feature_title: t('user_mailer.welcome.feature_control_title'), feature_text: t('user_mailer.welcome.feature_control'), text_first_on_desktop: true
```

Comment:
If we wanted to go one further here -- the `feature_title` and `feature_text` values are derivable from just the `key` -- we could pass in just the key and the final bool value to the partial and let it interpolate the i18n values.

Reply:
I was considering this but then i18n-tasks wouldn’t be able to keep track of these strings so I’m not sure if it’s worth it.